AABB_tree/include/CGAL/AABB_traits.h

```diff
@@ -414,26 +416,38 @@ class AABB_traits
           CGAL::SMALLER : CGAL::LARGER;
       }
 
-    CGAL::Comparison_result operator()(const Point& p, const Bounding_box& bb, const Point& bound, Tag_true&) const
+      CGAL::Comparison_result operator()(const Point& p, const Bounding_box& bb, const Point& bound, const Tag_true&) const
```

Comment:
```suggestion
      CGAL::Comparison_result operator()(const Point& p, const Bounding_box& bb, const Point& bound, Tag_true) const
```
Pass tags by value: they are empty objects. If you pass them by reference (to const or not), I do not think the compiler can optimize as easily.

Reply:
So this is done ??